node/src/components/consensus/era_supervisor/tests.rs

```diff
@@ -298,6 +298,8 @@ async fn propose_and_finalize(
     block
 }
 
+// TODO Do we want to completely remove this test?
+#[ignore]
```

Comment:
This seems like something that should be resolved by deleting the test?

Reply:
Yes, but a conclusive decision was not reached, so i left it in, @afck should I just remove it then?

Reply:
I'm in favor of removing the test. (And everything that becomes unused due to it — we can always resurrect `MockProto` from git history, if needed.)
But we can also `#[ignore]` it for now and do it in a separate PR; might be a bigger change.

Reply:
FYI I removed the test in https://github.com/CasperLabs/casper-node/pull/638.